docs/Formula-Cookbook.md

```diff
@@ -91,7 +91,7 @@ Try to summarise from the [`homepage`](https://rubydoc.brew.sh/Formula#homepage%
 
 **We don’t accept new formulae into Homebrew/homebrew-core without a [`license`](https://rubydoc.brew.sh/Formula#license-class_method)!**
 
-Find the license identifier from the [SPDX License List](https://spdx.org/licenses/). We only accept licenses from this list that are a [Debian Free Software Guidelines license](https://wiki.debian.org/DFSGLicenses).
+Find the license identifier from the [SPDX License List](https://spdx.org/licenses/) (or `:public_domain` for software in the public domain). We only accept licenses from this list that are a [Debian Free Software Guidelines license](https://wiki.debian.org/DFSGLicenses) (or public domain based on [the Debian Free Software Guidelines on Public Domain software](https://wiki.debian.org/DFSGLicenses#Public_Domain)).
```

Comment:
```suggestion
Find the license identifier from the [SPDX License List](https://spdx.org/licenses/). We only accept licenses from this list that are a [Debian Free Software Guidelines license](https://wiki.debian.org/DFSGLicenses). For software in the public domain based on the [DFSG Guidelines on Public Domain software](https://wiki.debian.org/DFSGLicenses#Public_Domain), use `:public_domain` as the identifier.
```

Reply:
I initially had something like this but I thought the `Find the license identifier from the [SPDX License List]` was a bit confusing as a sentence uncaveated.

Reply:
The multiple parentheticals are awkward IMO. Perhaps flipping it around would work:

```suggestion
We only accept formulae that use a [Debian Free Software Guidelines license](https://wiki.debian.org/DFSGLicenses), or are released into the public domain following [DFSG Guidelines on Public Domain software](https://wiki.debian.org/DFSGLicenses#Public_Domain).

Use the license identifier from the [SPDX License List](https://spdx.org/licenses/) e.g. `license "BSD-2-Clause"`, or use `license :public_domain` for public domain software.
```